cmd/main.go

```diff
@@ -31,8 +31,8 @@ import (
 var globalFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "config-dir, C",
-		Value: getConfigDir(),
-		Usage: "Path to configuration directory.",
+		Value: getDefaultConfigDir(),
+		Usage: "Path to configuration directory. Must be provided, if unable to get home directory",
```

Comment:
`if unable to get home directory` is no useful to the user.  The user has no clue why home dir is not able to be found.  This message needs to be specific depending on `getDefaultConfigDir()` result.